components/metrics/METADATA

```diff
@@ -0,0 +1,16 @@
+name: "metrics"
+description:
+  "Subtree at components/metrics."
+
+third_party {
+  url {
+    type: GIT
+    value: "https://chromium.googlesource.com/chromium/src/components/metrics"
+  }
+  version: "734293972a4b3f6d949e489ac02145538fb59c70"
+  last_upgrade_date {
```

Comment:
We want to import it from 2018 instead of a more recent version?

Reply:
Yes as it has to match our base/metrics package, which itself is from 2018.

Reply:
and third_party/metric_proto